nodejs/index.js

```diff
@@ -25,26 +24,25 @@ app.use(bodyParser.json());
 app.post('/messages', function(req, res) {
   console.log('webhook PAYLOAD:\n', JSON.stringify(req.body, null, 4));
 
-  const appUserId = req.body.appUser._id;
+  const conversationId = req.body.conversation._id;
+  const appId = req.body.app._id;
+
   // Call REST API to send message https://docs.smooch.io/rest/#post-message
```

Comment:
nit: could update it to the final expected URL (same with the other link on line 23)

Reply:
Updated